### PR TITLE
Add l1 media to ecosystem apps section

### PR DESCRIPTION
**Pull request by DzhideX**
Added [l1.media](https://l1.media/) to the carousel below featured apps in the "Apps built on Joystream" section. 

Note: I've decided it wasn't necessary to implement the whole carousel functionality still as we only have one app (and it is not obvious that there are more in line waiting to be added) so this was a quick way and it can be expanded later. But, tagging @toiletgranny to make sure he agrees as well :)

### CODE REVIEW

**Comment by toiletgranny** [comment]
Looking good, thanks for tagging me! A few minor things:

1. Seems like the border around the app icon goes _outside_, could we switch that? This could be achieved through either a `box-shadow` or by adding a `:before` element.
2. Ideally, clicking on the app card opens a new tab. Currently, the link is opened within the same tab.
3. Seeing this, I realized I wasn't really expecting us to launch this carousel with only one app. Now that it's a reality, the empty space next to this l1.media's card looks a bit awkward and I was thinking if this would make sense for us to add a few placeholders to mitigate that. @DzhideX If you like the idea, I believe implementing this could be as simple as adding three boxes with auto width and height and then adding the skeleton layout as a single SVG image that [you can export out of Figma in here](https://www.figma.com/file/3uoobZkRhFF4yibFjCWa2D/Joystream-%7C-Website?node-id=7834%3A403649&t=3pdVGFPYhRrHMG8U-4) (it's the "mockup" frame). I believe it only makes sense to display these placeholder cards on larger screen sizes.

   image

**Comment by DzhideX** [comment]
Thank you for taking a look. I feel much better knowing you did :)

1. Tried initially with box-shadow but it wasn't working. Found [this](https://stackoverflow.com/questions/11583947/boxshadow-inset-not-working) (TIL!). I ended up going with the `::before` approach. Let me know if you think it still needs some adjusting!

2. I remember @dmtrjsg mentioned in discord that he didn't like `about:blank` appearing in the URL when opening a new tab and going to a link (which is a side effect of instructing the anchor element to open a new blank tab AFAIK). I don't mind either way.

3. So, here I definitely agree with the sentiment but the implementation was a bit finicky. To make sure it works well I had to adapt the width of the items + the gap between them (also had to shorten the text of l1media). Let me know if this looks good to you this way (and if you have any idea to make it better)!

**Comment by toiletgranny** [comment]
1. Interesting, did not realize that either! Glad the `::before` worked tho, looks great now!
2. Ah, I see. Since the `about:blank` only appears there for a split of a second, I never saw that as an issue. To me, the current implementation looks great, and I would recommend having it this way.
3. Thank you for giving it a try! Actually, my idea was only to export the _contents_ of the box, not the box in its entirety. This would allow us to have a fluid width for each "placeholder" box (see the video) But looking at my previous comment now, I can see how I did not make that particularly clear, so it's totally on me. The current implementation is a million times better already, in my opinion, and I think we can stick with it.

https://user-images.githubusercontent.com/35307309/212837051-4f2d9d81-f40d-4c8b-84d7-1a724fb1604f.mp4

**Comment by traumschule** [comment]
Pointing out the obvious

apps-grid

Please try new blurb (note line-breaks):

> l1.media for education, news, science, entertainment beyond borders.
> First community-run gateway by founding members implementing new features and funding models empowering creators.
> A unique place for your creativity.

**Comment by DzhideX** [comment]
> Ah, I see. Since the about:blank only appears there for a split of a second, I never saw that as an issue. To me, the current implementation looks great, and I would recommend having it this way.

Sounds good, let's leave it this way then :+1:

> Thank you for giving it a try! Actually, my idea was only to export the contents of the box, not the box in its entirety. This would allow us to have a fluid width for each "placeholder" box (see the video) But looking at my previous comment now, I can see how I did not make that particularly clear, so it's totally on me.

Oh wow, totally missed it but that's a much better idea. Implemented it this way so let me know @toiletgranny if it looks right and I think we can ship this!

> Pointing out the obvious...

That suggestion goes a bit against the design Adam had in mind and that's present in Figma. There's always the option of adding l1.media to the featured section (i.e., have it be in a section like Gleev underneath it) but it requires more assets which is why I believe we went with this approach. I'm also not sure if there are any other requirements for this but we could iron this out with Dmitry/Bedeho.

**Comment by dmtrjsg** [comment]
Re featured section content @bedeho is best to advise.

**Comment by dmtrjsg** [comment]
@traumschule @DzhideX @toiletgranny 

Let's proceed with adding to the featured section before the carousel can be populated fully (4 apps minimum) https://discord.com/channels/811216481340751934/1064557129639411733/1065571145304244254

File: src/locales/en/landing.json [hunk 1]
```diff
@@ -37,12 +37,19 @@
...
-        "info": "We've just launched, so our app collection is yet to grow. Check back soon for more exciting apps built on Joystream."
+        "info": "We've just launched, so our app collection is yet to grow. Check back soon for more exciting apps built on Joystream.",
+        "gleev": {
+          "name": "Gleev",
+          "description": "Gleev has its sole focus on the Web3 and Crypto content vertical, bringing together the like-minded creators and viewers around the shared passion of the present and the future of Crypto affairs.",
+          "platforms": "Web"
+        },
+        "l1Media": {
+          "name": "l1.media",
+          "description": "For education, news, science, entertainment beyond borders. First community-run gateway by founding members implementing new features and funding models empowering creators.",
```
Comment (traumschule) [inline src/locales/en/landing.json:50]:
```suggestion
          "description": "For education, news, science, entertainment beyond borders. First community-run gateway by founding members implementing new features and funding models to empower creators.",
```

**Review by traumschule — CHANGES_REQUESTED** [review]
Thanks!

**Review by traumschule — APPROVED** [review]
:rocket:

**Pull request merged**